docs/version0.28.md

```diff
@@ -44,6 +45,37 @@ To learn more about support for OpenJ9 releases, including OpenJDK levels and pl
 
 JITServer technology is now a fully supported feature on Linux&reg; on x86 and Linux on IBM Power&reg; systems (64-bit only). This feature remains a technical preview for Linux on IBM Z&reg; systems (64-bit only). For more information, see [JITServer technology](jitserver.md).
 
+### ![Start of content that applies to Java 17 plus](cr/java17plus.png) New JDK 17 features
+
+The following features are supported by OpenJ9:
+
+- [JEP 409](https://openjdk.java.net/jeps/409): Sealed Classes
+    - Promoted from preview to a production-ready feature in this release.
+- [JEP 411](https://openjdk.java.net/jeps/411): Deprecate the Security Manager for Removal
+
+The following features will be supported by OpenJ9 in a future release:
+
+- [JEP 391](https://openjdk.java.net/jeps/391): macOS/AArch64 Port
+- [JEP 412](https://openjdk.java.net/jeps/412): Foreign Function & Memory API (Incubator)
+- [JEP 414](https://openjdk.java.net/jeps/414): Vector API (Second Incubator)
```

Comment:
I think the safe answer is to say we don't support it. @gita-omr your thoughts?

Reply:
We do support it. It's just that we only accelerate a small subset of vector operations. Currently we accelerate `load/store/add/mul/sub` without masks for all species and also `div` (without masks) for floating point species.

Reply:
Those methods should run around 10 times faster than they did in previous release.

Reply:
> we only accelerate a small subset of vector operations

I can add a note. On which platforms?

Reply:
On all platforms

Reply:
I only measured on Power but the code is common. I will try to confirm soon.